Easydict/objc/ViewController/Window/BaseQueryWindow/EZBaseQueryViewController.m

```diff
@@ -626,6 +636,18 @@ - (void)updateQueryTextAndParagraphStyle:(NSString *)text actionType:(EZActionTy
     self.queryModel.actionType = queryType;
 }
 
+- (void)updateActionType:(EZActionType)actionType {
+    self.queryModel.actionType = actionType;
+}
+
+- (void)showTipsView:(BOOL)show {
+    // judge if show tips view
+    self.showTipsView = show;
+    // update view
+    [self resetQueryAndResults];
```

Comment:
This code is bad, if query window has result, I just show the window again, it will clear query text and result, this is a bad action.